src/App/App.jsx

```diff
@@ -1,24 +1,22 @@
-/* eslint-disable */
 import React from "react";
-import {BrowserRouter as Router, Route} from "react-router-dom";
+import { BrowserRouter as Router, Route } from "react-router-dom"; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Please don't disable eslint if you don't know how to solve it. Try solving it by reading up the documentation. If you still can't solve it. Please ask for help